src/Core/src/Fonts/FontManager.Windows.cs

```diff
@@ -37,15 +44,42 @@ public FontManager(IFontRegistrar fontRegistrar, IServiceProvider? serviceProvid
 		{
 			_fontRegistrar = fontRegistrar;
 			_serviceProvider = serviceProvider;
+
+			Application.Current.Resources.RegisterPropertyChangedCallback(Control.FontFamilyProperty, OnPropertyChanged);
+			Application.Current.Resources.RegisterPropertyChangedCallback(Control.FontSizeProperty, OnPropertyChanged);
```

Comment:
The docs for `RegisterPropertyChangedCallback` says

> Registers a notification function for listening to changes to a specific [DependencyProperty](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.dependencyproperty?view=winrt-22621) on this [DependencyObject](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.dependencyobject?view=winrt-22621) instance.

I have never used this, but is it maybe saying that it is waiting for the `FontFamily` property to change on the `Resources` object?

Reply:
If people want to change the styles, they can set things in the WinUI App.xaml and then maui will always use that.